doc/conf.py.in

```diff
@@ -158,6 +158,7 @@ man_pages = [
     ('misc/installroot.7', 'dnf5-installroot', 'Installroot Parameter', AUTHORS, 7),
     # TODO(jkolarik): ('misc/modularity.7', 'dnf5-modularity', 'Modularity Overview', AUTHORS, 7),
     ('misc/specs.7', 'dnf5-specs', 'Patterns Specification', AUTHORS, 7),
+    ('misc/system-state.7', 'dnf5-system-state', 'System state', AUTHORS, 7),
```

Comment:
We still need to add it to `doc/CMakeLists.txt` and the spec file.

Reply:
Done.